### DISCUSSION TITLE
An issue with orbit/target translations based on camera-change event from model-viewer

### DISCUSSION THREAD

**Discussion by karanganesan** [google/model-viewer#3347] [Q&A]
### Description

Hi, not sure what exactly is the bug, but I have created a glitch url for the same
code - https://glitch.com/edit/#!/leaf-necessary-toad?path=index.html
live - https://leaf-necessary-toad.glitch.me/

so you can move in 1st model-viewer on top, I have added listener for camera change on viewer #1 and get the model orbit and target and apply it to viewer #2

it's not translating 1:1

#### Live Demo
https://leaf-necessary-toad.glitch.me/

### Version
- model-viewer: latest

### Browser Affected
- [x] Chrome, version: 100.0.1185.29
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS
- [ ] Android
- [ ] iOS
- [ ] Linux
- [x] MacOS
- [ ] Windows

### This is how I translate the values from one viewer to another 
```javascript
const v1 = document.querySelector("#viewer-1");
const v2 = document.querySelector("#viewer-2");
v1.addEventListener("camera-change", () => {
    const orbit = v1.getCameraOrbit();
    const target = v1.getCameraTarget();
    v2.setAttribute("camera-orbit", orbit.toString());
    v2.setAttribute("camera-target", target.toString());
    v2.jumpCameraToGoal(); // maybe not needed?
    v2.updateFraming(); // maybe not needed?
});
```

### Video of the issue
<video src="https://user-images.githubusercontent.com/6049925/162012027-3402e19e-dcd7-465d-bb79-7f05d4e60e9d.mov
" width="100%"/>

**Comment by elalish** [comment]
Our zoom feature changes both camera radius and field of view; you just forgot to set the second one.

**Comment by karanganesan** [comment]
hey @elalish, tried added fov also, still I see the issue...

<img width="50%" alt="Screen Shot 2022-04-07 at 8 31 51 AM" src="https://user-images.githubusercontent.com/6049925/162111829-9839de78-962f-481f-b840-a4a367d3fe34.png">

**Reply by karanganesan** [reply]
```javascript
v1.addEventListener("camera-change", () => {
    const orbit = v1.getCameraOrbit();
    const target = v1.getCameraTarget();
    const fov = v1.getFieldOfView();
    v2.setAttribute("camera-orbit", orbit.toString());
    v2.setAttribute("camera-target", target.toString());
    v2.setAttribute("field-of-view", `${fov}deg`);
});
```

**Reply by elalish** [reply]
Those two images look identical to me now; what issue do you see?
